app/Http/Controllers/v1/CategoryController.php

```diff
@@ -0,0 +1,76 @@
+<?php
+
+namespace App\Http\Controllers\v1;
+
+use App\Http\Controllers\Controller;
+use App\Models\Category;
+use Illuminate\Http\Request;
+use Illuminate\Http\Response;
+
+class CategoryController extends Controller
+{
+    public function index()
+    {
+        $categories = Category::all();
+
+        return response()->json($categories, Response::HTTP_OK);
```

Comment:
We return API responses with Laravel's `Resource` type. You can do `php artisan make:resource CategoryResource`

You can look at Chris' latest PR for reference, but you are essentially defining the structure of exactly what you want to return. [here](https://github.com/chrissantillan/UnlockEdv2/blob/17bbac8fb3ecad79bbbab65ab0468443e3148aa7/app/Http/Resources/ProviderPlatformResource.php)

Then you can return things from your controllers by doing `CategoriesResource::collection($categories)`

Also you can do `Category::all()->paginate(10);`